docs/src/app/pages/codibotintro.md

```diff
@@ -0,0 +1,35 @@
+# Platform: MIT App Inventor Codi Bot
+
+The MIT App Inventor Codi Bot is a hands-on IoT kit. Our adorable mascot can be controlled via App Inventor through Bluetooth communication. This educational kit is easy to use and guides users through the whole process of development, from building a robot to programming it. 
+
+<div style="text-align: center; font-size: 75%; margin: 16pt 0;">
+![Image of the MIT App Inventor Codi Bot.](/assets/howtos/codibot1.jpg)
+
+</div>
+
+## Parts list (<a href="https://www.robotkingdom.com.tw/product/mit-app-inventor-codi-bot/" target="_blank">Purchase link)</a>:
```

Comment:
Move the closing parenthesis outside of the `</a>`.

Reply:
ok